js_modules/dagit/packages/core/src/runs/RunTimeline.tsx

```diff
@@ -481,7 +442,7 @@ const RunTimelineRow = ({
           return (
             <RunChunk
               key={batch.runs[0].id}
-              $color={mergeStatusToColor(batch.runs)}
+              $background={mergeStatusToBackground(batch.runs)}
```

Comment:
It looks like this `RunTimelineRow` component would memoize cleanly, maybe it's worth doing now that there's some pretty fancy logic happening in here? (Or we could pull background out to a useMemo, but I feel like memoizing the entire component might be better?)

Reply:
Good thought. The `range` prop on `RunTimelineRow` would change pretty regularly, as the start/end window shifts with every poll result. I'd also worry slightly about memoizing the background value, since it would be memoizing based on an array of run objects. I think the array shouldn't change too often, but I also think most of the batches that get chunked together won't be prohibitively large, so I'm not sure it's worth memoizing.

Let's try it in a heavy timeline and see how the rendering costs look, and I'll add some memos in a followup if it looks worthwhile.